pkg/test-infra/tiflash/operation.go

```diff
@@ -0,0 +1,203 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tiflash
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ngaut/log"
+	"github.com/pingcap/errors"
+	clusterTypes "github.com/pingcap/tipocket/pkg/cluster/types"
+	"github.com/pingcap/tipocket/pkg/test-infra/fixture"
+	"github.com/pingcap/tipocket/pkg/test-infra/tidb"
+	"github.com/pingcap/tipocket/pkg/test-infra/util"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Ops knows how to operate TiDB and TiFlash on k8s
+type Ops struct {
+	cli client.Client
+	*tidb.TidbOps
+}
+
+// New creates Ops
+func New(cli client.Client, tidbClient *tidb.TidbOps) *Ops {
+	return &Ops{cli, tidbClient}
+}
+
+// Apply TiFlash cluster
+func (o *Ops) Apply(tc *Recommendation) error {
+	// config file initialization
+	if fixture.Context.TiKVConfigFile == "" {
+		fixture.Context.TiKVConfigFile = "/config/tiflash/tikv.toml"
+	}
+	if fixture.Context.PDConfigFile == "" {
+		fixture.Context.PDConfigFile = "/config/tiflash/pd.toml"
```

Comment:
Setting fixture.Context here isn't recommended, and "/config/tiflash/pd.toml" will confuse someone unfamiliar with Tipocket.

Reply:
How about rename the configmap dir to config on this PR?